zulipterminal/ui_tools/boxes.py

```diff
@@ -37,6 +37,7 @@ def private_box_view(self, button: Any=None, email: str='') -> None:
             email = button.email
         self.to_write_box = ReadlineEdit(u"To: ", edit_text=email)
         self.msg_write_box = ReadlineEdit(multiline=True)
+        self.model.controller.view.set_footer_text(user_state='msg_compose')
```

Comment:
We could add a listener for user_state. Whenever we update like `view.user_state` footer text changes accordingly and we don't have to do it manually.

Reply:
Great, could you point me to some resources for the same so I can get started on this?

Reply:
Sure! https://www.geeksforgeeks.org/python-property-function/

Reply:
@amanagr, tried doing that. Is this what you were suggesting?

Reply:
Yes, This will help when you will use other user_states.